packages/ocaml-sat-solvers/ocaml-sat-solvers.0.8/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "An abstraction layer for integrating SAT Solvers into OCaml"
+description: "An abstraction layer for integrating SAT Solvers into OCaml."
+maintainer: ["Oliver Friedmann" "Martin Lange"]
+authors: ["Oliver Friedmann" "Martin Lange"]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/ocaml-sat-solvers"
+bug-reports: "https://github.com/tcsprojects/ocaml-sat-solvers/issues"
+depends: [
+  "ocaml" {>= "4.8"}
+  "dune" {>= "3.10"}
+  "minisat" {>= "0.4"}
+  "z3" {>= "4.8.11"}
```

Comment:
The latest version I build z3 on my mac was z3.4.13.2, both on OCaml 4.14.1 and OCaml 5.3.0. Is bumping the bound to `4.13.2` okay?

Reply:
Thanks for pointing this out. I should find a machine where older versions of z3 are building and figure out which ones are not compatible with ocaml5. It seems that the issue is with z3, though. I'd leave the lower bound as it is until we can test it